docs/application/tizen-studio/index.md

```diff
@@ -1,45 +1,45 @@
 # Tizen Studio
 
-**Tizen Studio 3.0 is now launched!**&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![Download](media/ic_docs_download.png)  **Download**](https://developer.tizen.org/development/tizen-studio/download)
+**Tizen Studio 3.0 is now launched!**&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![Download](media/ic_docs_download.png) **Download**](https://developer.tizen.org/development/tizen-studio/download)
 
-The Tizen Studio platform is based on a new version of Eclipse, so you can experience a stable, lightweight, and consistent user interface that is unique to the Tizen development environment. With the Tizen Studio, you can select and install only those platforms and profiles you need.
+The Tizen Studio platform is based on a new version of Eclipse, so you can experience a stable, lightweight, and consistent user interface that is unique to the Tizen development environment. Using Tizen Studio, you can select and install only those platforms and profiles you need.
 
-In the Tizen Studio, you can perform all the same development activities that were possible with the old Tizen SDK: you can develop, build, debug, profile, and emulate Tizen native and Web applications. The new development environment in the Tizen Studio is simply better. It allows you to develop applications more comfortably and is more focused on Tizen development. There are about 200 online sample applications available to increase your understanding of Tizen applications, and the sample library continues to grow.
+Similar to the old Tizen SDK, in Tizen Studio, you can perform all the development activities like: develop, build, debug, profile, and emulate Tizen native and Web applications. The new development environment in the Tizen Studio is simply better. It allows you to develop applications more comfortably and is more focused on Tizen development. There are about 200 online sample applications available to increase your understanding of Tizen applications, and the sample library continues to grow.
```

Comment:
@journey2w How about removing the phrase related with Tizen SDK from this sentence? For example, as like 
"In Tizen Studio, you can perform all the development activities like: develop, build, debug, profile, and emulate Tizen native and Web applications. It also allows you to develop applications more comfortably. There are about 200 online sample applications available~"